### ISSUE TITLE
Calendar is not usable at 200%

### ISSUE THREAD

**Issue by Gabrielaparada** [duetds/date-picker#128]
The calendar does not support 200% zoom which fails WCAG assertion 1.4.4. 
Users should be able to access content and functionality when the text size is increased by 200%. This is particularly important for low vision users as it can allow them to make content more readable. No loss of functionality should occur at 200% text size. 

**To Reproduce**
Open the calendar on your phone and proceed to apply zoom text, the calendar will become unusable at around 150%. Users are not able to reach the off-screen elements.

**Expected behavior**
The calendar should be usable up to 200%

**Screenshots**
**At 150% the 'Next' button and the Sunday column are not visible or reachable**
![IMG_2544](https://github.com/duetds/date-picker/assets/53956875/1aadc152-54cc-4296-b097-2a75c98f629f)

**At 200%  most elements are not fully visible**
![IMG_2545](https://github.com/duetds/date-picker/assets/53956875/56898a1e-07e9-4f0d-ad2e-bdf45ef8ab68)

https://github.com/duetds/date-picker/assets/53956875/5c396a57-ffd0-4d49-a30b-5c128597a12a

Tested on:
 - Device: iPhone 13 pro
 - OS: ios 17.4.1
 - Browser: Safari
